src/foundation/Mixins.ts

```diff
@@ -1,17 +1,26 @@
 /* External dependencies */
 import type { CSSProperties } from 'react'
 import { isNil } from 'lodash-es'
+import type { SimpleInterpolation } from 'styled-components'
 
 /* Internal dependencies */
 import { css } from './FoundationStyledComponent'
 
-export const absoluteCenter = (otherTransforms: any) => `
+export const absoluteCenter = (otherTransforms: SimpleInterpolation) => `
   position: absolute;
   top: 50%;
   left: 50%;
   transform: translate(-50%, -50%) ${otherTransforms};
 `
 
+export const centeredBackgroundImage = (width?: number, height?: number, imageUrl?: string) => `
+  background-image: url(${imageUrl});
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: ${width}px, ${height}px;
+  background-clip: content-box;
+`
```

Comment:
```suggestion
export const centeredBackgroundImage = <Type extends number | string>(imageUrl: string, ...sizes: [Type] | [Type, Type]): ReturnType<typeof css> => {
  const [width, height = width] = sizes
  return css`
    background-image: url(${imageUrl});
    background-position: center;
    background-repeat: no-repeat;
    background-size: ${width}px, ${height}px;
    background-clip: content-box;
  `
}
```

반드시 필요한 매개변수는 필수로 하는 게 좋을 거 같아요. 옵셔널로 두게 되면 잘못된 css 속성을 반환하게 됩니다.

Reply:
타입 깎는 에드